packages/CalDAV-Core.package/CALMonthPanel.class/instance/buildDaysOfMonthWith..st

```diff
@@ -4,6 +4,6 @@ buildDaysOfMonthWith: aBuilder
 	^ aBuilder pluggablePanelSpec new
 		model: self;
 		name: 'daysOfMonth';
-		frame: (0 @ self buttonHeight corner: 1 @ 1);
+		frame: (0 @ (self buttonHeight + 1/6) corner: 1 @ 1);
```

Comment:
should we put magic number (1/6) in a variable?

Reply:
We could? That's basically the additional space so the weekdays fit between the buttons on top of the month view and the days of the month at the bottom. So maybe spaceForWeekdays?

Reply:
sounds good